src/zentables/zentables.py

```diff
@@ -519,7 +519,7 @@ def _internal_freq_table(
             mask = _do_suppression(pivot, low, high)
 
         if props is None and suppress:
-            return pivot.where(~mask).fillna("*")
+            return pivot.where(~mask).astype(object).fillna("*")
```

Comment:
Casting to type `object` seems like it's probably wrong?

Reply:
Completely true. This was a bad call.